weave

```diff
@@ -1795,18 +1814,40 @@ case "$COMMAND" in
     # intentionally undocumented since it assumes knowledge of weave
     # internals
     create-bridge)
+        cat 1>&2 <<EOF
```

Comment:
That is a lot of output. Perhaps we should only show it when docker isn't running?